Ednpoints.txt

```diff
@@ -0,0 +1,132 @@
+The same routes for users, chefs and delivery
+POST: /auth/register - registration
+POST: /auth/login - enter to account
+GET: /auth/logout - exit from account
+POST: /auth/forgot-password - request to change password
+POST: /auth/reset-password - password reset
+
+/************/
+
+USER ENDPOINTS:
+
+GET: /users/:userId - get user's info
+PATCH: /users/:userId - update user's info
+DELETE: /users/:userId - delete user's account
+
+GET: /dishes - get a list of dishes
+GET: /dishes/:dishId - get dish information
+GET: /dishes/popular - get popular dishes
+GET: /dishes/random - get some random dish
+
+GET: /chefs - get all chefs
+
+GET: /users/:userId/favourites?type=chef(dish) - get user's favourite dishes or chefs
```

Comment:
I like using `/users/:userId/favorites/:type` with further Joi validation of type value more
query parameters are used for pagination and querying purposes mostly

Reply:
Okay, that's clear. Let's do as you suggest